src/Shell/Task/MigrationSnapshotTask.php

```diff
@@ -246,7 +246,15 @@ public function fetchTableName($className, $pluginName = null)
                 $tables[] = $table->associations()->get($key)->_junctionTableName();
             }
         }
-        $tables[] = $table->table();
+        $t = $table->table();
+        $splitted = array_reverse(explode('.', $t));
```

Comment:
I think @makallio85 wants to cover the case where you define your table name as ``schema.table``.